iib/workers/config.py

```diff
@@ -29,6 +29,7 @@ class Config(object):
     # Don't allow the worker to fetch more messages than it can handle at a time. This is so that
     # other tasks aren't starved. This will only be useful once more workers are enabled.
     worker_prefetch_multiplier = 1
+    worker_total_retries = 5
```

Comment:
The `worker_`  prefix is reserved for the Celery worker library configuration. Please use the `iib_` prefix and alphabetize it.

Reply:
Could you please document this in the read me?